src/main/java/org/folio/circulation/support/fetching/MultipleCqlIndexValuesCriteria.java

```diff
@@ -3,22 +3,26 @@
 import static org.folio.circulation.support.http.client.CqlQuery.noQuery;
 
 import java.util.Collection;
+import java.util.function.BiFunction;
 
 import org.folio.circulation.support.results.Result;
 import org.folio.circulation.support.http.client.CqlQuery;
 
-public class MultipleCqlIndexValuesCriteria {
-  final String indexName;
-  final Collection<String> values;
-  final Result<CqlQuery> andQuery;
-
-  private MultipleCqlIndexValuesCriteria(String indexName,
-      Collection<String> values, Result<CqlQuery> andQuery) {
+import lombok.Builder;
+import lombok.Getter;
+import lombok.Singular;
 
-    this.indexName = indexName;
-    this.values = values;
-    this.andQuery = andQuery;
-  }
+@Builder
+@Getter
```

Comment:
Is it possible to make the getter's package private, as only the finder should really be using them?

Reply:
Getters has package/default access now.